### ISSUE TITLE
block_content doesn't render content inside tag

### ISSUE THREAD

**Issue by scottcorgan** [desiringgod/components-rails#3]
I have the code

```erb
<button class="<%= classes %> h-40">
  <% block_content %>
</button>
```

Then I use this like

```erb
<%= button do %>
  Stuff in my button
<% end %>
```

The html that gets rendered is

<img width="220" alt="screen shot 2017-08-03 at 3 09 18 pm" src="https://user-images.githubusercontent.com/974723/28941462-bfacf9da-785d-11e7-95ed-6061d9cf6322.png">

"Stuff in my button" should render in inside the button.

**Comment by joeosburn-desiringgod** [comment]
@scottcorgan I think this was just a typo, because your bug is real regardless, but just to be sure, your code should be:
```
<%= block_content %>
```

In any case, I'm going to work on solving this bug soon! Thanks for reporting it.

**Comment by scottcorgan** [comment]
@joeosburn-desiringgod thanks 👍

**Comment by joeosburn-desiringgod** [comment]
Resolved in #8 and #9
